wavefront_pyformance/runtime_metrics.py

```diff
@@ -0,0 +1,168 @@
+"""Python Runtime Metrics Collection Class."""
+
+import gc
+import multiprocessing
+import os
+import resource
+import threading
+
+import psutil
+
+
+class RuntimeCollector(object):
+    """Python Runtime Metrics Collection Class."""
+
+    # pylint: disable=E0012,R0205
+
+    def __init__(self, registry=None):
+        """Construct Runtime Metrics Collector."""
+        self.registry = registry
+        self.pid = os.getpid()
+        self.process = psutil.Process(self.pid)
+        self.pname = self.process.name()
+        self.status = self.process.status()
+        self.custom_tags = {"process_id": str(self.pid),
+                            "process_name": self.pname,
+                            "process_status": self.status}
+
+    def collect_cputimes(self):
+        """Collect CPU Times."""
+        cputimes = self.process.cpu_times()
+        system_cpu = cputimes[1]  # system cpu
+        metric1 = self.registry.gauge("cpu.times",
+                                      tags=self.custom_tags)
+        metric1.set_value(system_cpu)
```

Comment:
We could set the value right away after retrieving the gauge, e.g.
```
        self.registry.gauge("cpu.times",
                            tags=self.custom_tags).set_value(system_cpu)
```
here and further down

Reply:
WDYT?

Reply:
@ustinov Yes, that's better. Made the change.